compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1728,6 +1728,10 @@ rustc_queries! {
         desc { |tcx| "collecting child items of module `{}`", tcx.def_path_str(def_id) }
         separate_provide_extern
     }
+    query ambiguity_module_children(def_id: DefId) -> &'tcx [AmbiguityModChild] {
```

Comment:
A query is not necessary here, you can add a `fn ambiguity_module_children_untracked` method to `impl CStore` in `compiler\rustc_metadata\src\rmeta\decoder\cstore_impl.rs` instead.